uaclient/config.py

```diff
@@ -247,12 +247,14 @@ def _attached_service_status(
         self, ent, inapplicable_resources
     ) -> "Dict[str, str]":
         details = ""
+        userFacingDetails = ""
```

Comment:
Huh, surprised that our linting tools aren't picking this up as an invalid variable name. A nit, but this would be more consistent as `user_facing_details`.

Reply:
fixed. agreed, /me got wrapped up in our json field names.